### DISCUSSION TITLE
Disable the `Security` tab since it's not in use

### DISCUSSION THREAD

**Discussion by TreyWW** [saadeghi/daisyui#2918] [Ideas / feature requests]
Since DaisyUI doesn't use the Security Vulnerability section on github (i'm guessing since theres no chance of a security vulnerability here in the first place) I feel like it would be ideal to remove it from the top tab to remove unused clutter

![image](https://github.com/saadeghi/daisyui/assets/73353716/6559ea29-13ba-446f-a8fb-5790f699502a)
![image](https://github.com/saadeghi/daisyui/assets/73353716/5f824357-e7a3-4ef6-b2ca-5c1cb98df5ed)

**Comment by saadeghi** [comment]
Is there an option to remove the tab?

**Reply by TreyWW** [reply]
Ah, didn't think of that lol, looking through settings I can't find an open either. I guess you can close the discussion - sorry about that
